centos2almaconverter/actions/postgres.py

```diff
@@ -18,6 +19,42 @@ def _do_check(self):
         return not postgres.is_postgres_installed() or not postgres.is_database_initialized() or not postgres.is_database_major_version_lower(_ALMA8_POSTGRES_VERSION)
 
 
+class AssertPostgresLocaleMatchesSystemOne(action.CheckAction):
+    def __init__(self):
+        self.name = "checking if system locale is safe for Postgres databases upgrade"
+        self.description = """Postgres database upgrade expects system locale to match the one databases were created with.
+\tYou may need to change system locale (see /etc/locale.conf and the locale command) or
+\tdatcollate and datctype properties of Postgres databases to match each other."""
+        self.service_name = 'postgresql'
+
+    def _do_check(self):
+        config_path = os.path.join(postgres.get_data_path(), 'pg_hba.conf')
+
+        try:
+            files.backup_file(config_path)
+            files.push_front_strings(config_path, ["local template1 postgres trust #Added by Plesk\n"])
+            util.logged_check_call(['systemctl', 'reload-or-restart', self.service_name])
```

Comment:
NIT: It could be helpful to move this into the [systemd functions module](https://github.com/plesk/dist-upgrader/blob/main/pleskdistup/common/src/systemd.py)